integration/flink/shared/build.gradle

```diff
@@ -0,0 +1,182 @@
+/*
+/* Copyright 2018-2023 contributors to the OpenLineage project
+/* SPDX-License-Identifier: Apache-2.0
+*/
+
+import io.franzbecker.gradle.lombok.task.DelombokTask
+
+plugins {
+    id 'java'
```

Comment:
`java` plugin is applied unncessarily in the presence of the `java-library` plugin